blue_modules/mintlayer/mintlayer.js

```diff
@@ -61,14 +62,14 @@ export const getTxInput = async (outpointSourceId, index) => {
   return webviewEventBus.exec(wasmMethods.encode_input_for_utxo, [outpointSourceId, index]);
 };
 
-export const getOutputs = async (amount, address, networkType) => {
+export const getOutputs = async ({ amount, address, networkType }) => {
```

Comment:
getOutputs is more complex, refer to 
https://github.com/mintlayer/mojito-browser-extension/blob/main/src/services/Crypto/Mintlayer/Mintlayer.js#L166

Reply:
Done